app/controllers/list_controller.rb

```diff
@@ -58,6 +58,10 @@ class ListController < ApplicationController
 
       list = TopicQuery.new(user, list_opts).public_send("list_#{filter}")
 
+      if params[:category].blank? && filter == :latest && guardian.is_staff?
+        list.show_welcome_topic_banner = Post.show_welcome_topic_banner?
```

Comment:
I'm wondering if it is necessary to ship this as part of the topic list instead of adding it as a site attribute which is only fetched once while initialising the Ember app.